03/1_pointer/main.c

```diff
@@ -0,0 +1,39 @@
+#include <assert.h>
+#include <stdio.h>
+
+#include "algorithm.h"
+
+int main(void)

```

Comment:
this time let's use

#define LENGTH (5)

and use it for both array init and size_t